backend/src/data/entities/careerParh.ts

```diff
@@ -0,0 +1,14 @@
+import { Entity, PrimaryGeneratedColumn, ManyToOne } from 'typeorm';
+import { Domain } from './domain';
+
+@Entity()
+export class CareerParh {
```

Comment:
each entity should extend AbstractEntity if entity has `id`
```suggestion
export class CareerParh  extends AbstractEntity {
```

if it is many-to-many table - extend BaseEntity (from typeorm) and add `createdAt`, `updatedAt`, `deletedAt` columns